Library/OcAppleKernelLib/CommonPatches.c

```diff
@@ -60,13 +60,21 @@ PatchAppleCpuPmCfgLock (
   UINT8  *WalkerEnd;
   UINT8  *WalkerTmp;
 
+  if (Patcher == NULL) {
```

Comment:
Please move that below. Not found looks like an error, but we should rather see skipping. Similar to other places.

Reply:
https://github.com/acidanthera/OpenCorePkg/pull/359/commits/6ebf9c352c2ab60022bbe4717299dae6459de247